core/src/main/java/com/graphhopper/routing/profiles/RoadAccess.java

```diff
@@ -0,0 +1,51 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.routing.profiles;
+
+import com.graphhopper.util.Helper;
+
+public enum RoadAccess {
+    // order is important here as we assume "smaller index" means "broader access"
+    OTHER("other"), YES("yes"),
```

Comment:
Should other really be the first/broadest access? 

There are access values like [unknown, military, residents, and many undocumented values](https://taginfo.openstreetmap.org/keys/access#values).

Reply:
Yes, indeed.